### ISSUE TITLE
When a page's back button is clicked, go to the parent page

### ISSUE THREAD

**Issue by WANZARGEN** [cloudforet-io/cloudforet#85]
For consistency of the back button,
I would like to suggest how to apply the policy “When the back button of any page is clicked, it moves to the parent page”.

### As Is
1. There is no back button on the topmost page.
2. Go to the parent page in overlay page.
3. Works the same as browser's back button.

Browser's back button: If user directly access to a specific page of console, if the previous page the user stayed on is github, go to github.

### To Be
1. There is no back button on the topmost page. (same)
2. Go to the parent page in overlay page. (same)
3. **Works the same as in overlay page's back button. (hope for change)**

**Comment by WANZARGEN** [comment]
Target Pages:

Cloud Service Detail Page
Provider Add Page
Policy Create Page
Policy Detail Page
Role Create Page
Role Edit Page
User Manage Notification Page
Alert Detail Page
Cloud Service History Detail Page
Collector History Page
Collect Job Page
Collector Plugins Page
Collector Create Page
Service Account Add Page
Service Account Detail Page
Budget Create Page
Budget Detail Page
Budget Update Page
Cost Dashboard Create Page
Dashboard Create Page
Dashboard Manage Variables Page
Notice Create Page
Notice Detail Page
Notice Update Page
Notification Add Page
Project Detail Page
Project Alert Event Rule Page
Project Notification Add Page

**Comment by justankiit** [comment]
I would like to work on this! @WANZARGEN

**Comment by WANZARGEN** [comment]
> I would like to work on this! @WANZARGEN

Oh, thank you so much for contributing to resolving this issue. @kumarankit999 
This issue is related to the [console repository](https://github.com/cloudforet-io/console).
To address this issue, a composable called [`useGoBack`](https://github.com/cloudforet-io/console/blob/master/apps/web/src/common/composables/go-back/index.ts) has been added, but it hasn't been applied to all pages yet.
Applying this composable is a key task.
Additionally, many pages have been added or removed since the creation of this issue, so the target pages have also changed.
However, all page components follow the convention of being named with 'Page,' making them easy to find.
Currently, the `useGoBack` composable has been integrated into the `CollectorCreatePage`, `CollectorDetailPage`, and `DashboardCreatePage`.
It seems like we can use these pages as references to apply the composable to other pages.

Feel free to share any better ideas or suggestions at any time. It would be great to collaborate and make improvements together.
If you need any further explanations or assistance, feel free to leave a comment on the corresponding issue, and I'll be happy to help!

**Comment by justankiit** [comment]
Yes, Thanks @WANZARGEN, I think we can wait for the composable useGoBack. If it applies to all pages then we will be working on this. Any other Suggestions are welcome!!!!

**Comment by WANZARGEN** [comment]
The `useGoBack` composable has already been developed, and resolving this issue involves applying it to other pages (pages with a back button) beyond the three mentioned. 
Would you like assistance in applying the `useGoBack` composable to the pages to address this issue, @kumarankit999? 😄

**Comment by justankiit** [comment]
Ok!! I got it now @WANZARGEN !! I'm running it on my localhost

**Comment by WANZARGEN** [comment]
Oh, feel free to ask for help anytime if you encounter difficulties during the process, @kumarankit999 . :)

Your feedback on the contribution process is also welcome at any time. Since a systematic process for contributions hasn't been established yet, if you're facing challenges while contributing, others might be experiencing the same. Therefore, your feedback is highly valuable in improving our community ecosystem.

**Comment by justankiit** [comment]
I will definitely work on this and make contributions in this repo!
